exercises/grains/.meta/template.j2

```diff
@@ -0,0 +1,29 @@
+{%- import "generator_macros.j2" as macros with context -%}
+
+{% macro test_case(case) %}
+    {%- set input = case["input"] %}
+    {%- if case is error_case %}
+    def test_{{ case["description"] | to_snake }}(self):
+          with self.assertRaisesWithMessage(ValueError):
+                {{ case["property"] | to_snake }}({{ input["square"] }})
+    {%- else%}
+    def test_{{case["property"] | to_snake}}_{{ case["description"] | to_snake }}(self):
```

Comment:
Since we no longer test for parameters to `total()`, I think this can change as follows:
```suggestion
    def test_{{ case["description"] | to_snake }}(self):
```
...which should allow the template to be simplified a bit more.

Reply:
Made the swap to the function definition line, unified the cases so that the function definition occurs outside the conditional.